CharacterCreation/03_Feats.txt

```diff
@@ -200,6 +200,12 @@ head out and looking at your enemy. And somehow you still hit them (some of the
 Roll a d10 before any Blind Firing or Hip Firing action. If the roll is from 6 to 10, 
 Gain -2 miss chance for that firing action; If the roll is from 1 to 5, nothing happens.
 
+Last Stand:
+Prerequisite: Level 4
+Don't die alone. Always bring a friend. You may take an action to do one of 

```

Comment:
Please add some mention of how Last Stand is triggered (just to be crystal clear). Also, please add how many actions per turn they get since being "Downed" normally means you don't get any actions except saving throws (although that isn't stated specifically in the "Down" section of the basic rules). I.e., "If you are downed you may take 4-actions per turn to do the following (in addition to rolling your saving throw):"